gradle/libs.versions.toml

```diff
@@ -81,6 +82,7 @@ slf4j = "2.0.17"
 snowflake-jdbc = "3.23.2"
 spark34 = "3.4.4"
 spark35 = "3.5.5"
+spark40 = "4.0.1-SNAPSHOT"
```

Comment:
Will change to official released Spark 4.0 in the future.

Reply:
Shouldn't this be the RC now?

Reply:
I will change this to RC4.